R/Histogram-class.R

```diff
@@ -84,9 +84,9 @@ validate_Histogram <- function(x){
       stop("intervals must be ordered and nonoverlapping.", call. = FALSE)
     }
 
-    # 4. Intervals need to be non-overlapping
-    if(any(interval_start[2:(histogram_length)] < interval_end[1:(histogram_length-1)])){
-      stop("intervals must be ordered and nonoverlapping.", call. = FALSE)
+    # 4. Intervals need to be continuous
+    if(any(interval_start[2:(histogram_length)] != interval_end[1:(histogram_length-1)])){
+      stop("intervals must be continuous.", call. = FALSE)
```

Comment:
This is where the discussion from issue #112 could change the code. Dunno if you want to do that in the PR or a separate one.

Reply:
Slightly problematic to simply remove this code - will discuss in the issue itself.